.github/workflows/lint.yml

```diff
@@ -16,4 +16,8 @@ jobs:
         with:
           node-version: 18
 
-      - run: npx prettier --check .
+      - name: Pettier
+        run: npx prettier --check .
+
+      - name: Eslint
+        run: npx eslint .
```

Comment:
I might consider trying to add a version on these, because I think they're gonna download `eslint` and `prettier` every time.

Alternatively, you might want to add them as dev dependencies and run them with `npm exec`.

Reply:
Is that a problem? The check is much faster than the docker build.